code/modules/bitrunning/objects/gimmick_disks/dungeon_disk.dm

```diff
@@ -0,0 +1,226 @@
+
+/**
+ * Bitrunning gimmick loadouts themed around dungeon crawling.
+ * Mostly for fun, have niche but not insignificant advantages.
+ */
+/obj/item/bitrunning_disk/gimmick/dungeon
+	name = "bitrunning gimmick: dungeon crawling"
+	selectable_loadouts = list(
+		/datum/bitrunning_gimmick/alchemist,
+		/datum/bitrunning_gimmick/rogue,
+		/datum/bitrunning_gimmick/healer,
+		/datum/bitrunning_gimmick/wizard,
+	)
+
+
+/datum/bitrunning_gimmick/alchemist
+	name = "Alchemist"
+
+	granted_items = list(
+		/obj/item/clothing/suit/bio_suit/plaguedoctorsuit,
+		/obj/item/clothing/mask/gas/plaguedoctor,
+		/obj/item/clothing/head/bio_hood/plague,
+		/obj/item/storage/box/alchemist_basic_chems,
+		/obj/item/storage/box/alchemist_basic_chems,
+		/obj/item/storage/box/alchemist_random_chems,
+		/obj/item/storage/box/alchemist_chemistry_kit,
+	)
+
+/obj/item/reagent_containers/cup/bottle/alchemist_basic
+	name = "unlabeled bottle"
+	desc = "A small bottle. You don't remember what you put in it."
+
+	/// List of possible reagents we may pick from
+	var/static/list/possible_reagents = list(
+		/datum/reagent/aluminium, // Basic chems
+		/datum/reagent/bromine,
+		/datum/reagent/carbon,
+		/datum/reagent/chlorine,
+		/datum/reagent/copper,
+		/datum/reagent/consumable/ethanol,
+		/datum/reagent/fluorine,
+		/datum/reagent/hydrogen,
+		/datum/reagent/iodine,
+		/datum/reagent/iron,
+		/datum/reagent/lithium,
+		/datum/reagent/mercury,
+		/datum/reagent/nitrogen,
+		/datum/reagent/oxygen,
+		/datum/reagent/phosphorus,
+		/datum/reagent/potassium,
+		/datum/reagent/uranium/radium,
+		/datum/reagent/silicon,
+		/datum/reagent/sodium,
+		/datum/reagent/stable_plasma,
+		/datum/reagent/consumable/sugar,
+		/datum/reagent/sulfur,
+		/datum/reagent/toxin/acid,
+		/datum/reagent/water,
+		/datum/reagent/fuel,
+		/datum/reagent/toxin/leadacetate, // Funny chems
+		/datum/reagent/consumable/liquidgibs,
+		/datum/reagent/consumable/nutriment,
+		/datum/reagent/consumable/coffee,
+	)
+
+/obj/item/reagent_containers/cup/bottle/alchemist_basic/add_initial_reagents()
+	var/our_reagent = pick(possible_reagents)
+	reagents.add_reagent(our_reagent, 50)
+
+/obj/item/reagent_containers/cup/bottle/alchemist_random
+	name = "skull-labeled bottle"
+	desc = "A small bottle. You don't remember what you put in it."
+
+	/// List of random names this may have
+	var/static/list/possible_names = list(
```

Comment:
considering all of these end with bottle, we could as well only put adjectives in this list. It must be slightly obnoxious to write the same noun for each line after all.

Reply:
yeahhh holdover from an old system that did care, whoops, thanks